tensorboard/webapp/app_routing/internal_utils_test.ts

```diff
@@ -170,6 +171,110 @@ describe('app_routing/utils', () => {
     });
   });
 
+  describe('areSameRouteAndExperiments', () => {
+    it('returns true when both routes are null', () => {
+      expect(utils.areSameRouteAndExperiments(null, null)).toBeTrue();
+    });
+
+    it('returns false when one route is null', () => {
+      expect(utils.areSameRouteAndExperiments(buildRoute(), null)).toBeFalse();
+      expect(utils.areSameRouteAndExperiments(null, buildRoute())).toBeFalse();
+    });
+
+    it('returns false when different ', () => {
+      expect(utils.areSameRouteAndExperiments(buildRoute(), null)).toBeFalse();
+      expect(utils.areSameRouteAndExperiments(null, buildRoute())).toBeFalse();
+    });
```

Comment:
What makes this test case different than the previous one?

Reply:
I think I meant for this test to be 'returns false when different kind' but never got around to finishing it. Fixed it up now.